### ISSUE TITLE
Error: Deposit address has been spent from

### ISSUE THREAD

**Issue by Nacirema** [rajivshah3/IOTA-Balance-Finder#8]
Thank you for writing this program. It did find my missing 10 Iota (my IOTA Light Wallet 2.5.7 says I have 0 Iota). I entered the old seed and a new seed for your program but the 10 Iota I have were not sent to a new address. Instead, there was an error message: "Error: Deposit address has been spent from." So I altered the new seed and tried again and then I had the same error. This is a seed that is generated by me and not some other seed generator software. Do I need to keep changing the new seed until a deposit is successful? Also, if the deposit is successful does the program display the new address?

**Comment by albertoc81** [comment]
Same problem here. "Error: Deposit address has been spent from".  Help would be really appreciated.

**Comment by moneropezo** [comment]
The issues seems to be that this tool only indexes from the snapshot file. If you made a transaction after the snapshot happened, your IOTA will currently not be discovered by the tool. The new address where the IOTA is is not the one from the snapshot.

**Comment by lacksfish** [comment]
The issues seems to be that this tool only indexes from the snapshot file. If you made a transaction after the snapshot happened, your IOTA will currently not be discovered by the tool. The new address where the IOTA is is not the one from the snapshot.

**Comment by lacksfish** [comment]
**This could rediscover your lost IOTA if you made (a) transaction(s) after the snapshot happened.**

The simplest, recommended approach is to get the latest IOTA wallet from https://github.com/iotaledger/wallet/releases (v2.5.7)

Initialize your wallet with your IOTA seed.

Then, generate as many addresses as necessary (do not forget to re-attach every single one of them!). Don't give up, generate more than 100 addresses if needed. The more you've used your wallet in the past, the more addresses you will need to generate.

If that helped you recover, I'd be grateful for a donation so I can buy some salmon: 
>QBOMCBHQNQESDJNUYUUMPGNOCHVLQ9AQ9UYEFQASVJOBTBUZXEZWJQAXHGZOS9MUADSMVLCMDZGRMDTPWFL9IRDFXX

**Comment by Nacirema** [comment]
Wow. After all this time. It worked with the second address. I don't remember generating a second address before. Thank you, lacksfish. It was only 10 IOTA. I sent some to you. It won't buy much salmon, I'm afraid, unless you can wait for the IOTA value to go back up.

**Comment by ratzi17** [comment]
Same problem here. "Error: Deposit address has been spent from". But sorry, after initializing  my wallet with my IOTA seed, i am unable to generate new addresses (i am using IOTA light wallet 2.5.7). please help

**Comment by Nacirema** [comment]
I do not know about being unable to generate a new address as in the program not being able to do so. I thought there was not a way to generate a new address but clicked on RECEIVE and saw the first address and in a box was the word ATTACHED (or something similar) and when I clicked on that word what appeared were the words GENERATE NEW ADDRESS. I clicked that to generate a new address and then clicked ATTACH TO TANGLE. However, in HISTORY it is still pending. So, every 30 minutes I click on "Show bundle" in history and if it says REATTACH I click that. Eventually the word "Pending" should become "Confirmed."  I had to use history and the "Show bundle" and REATTACH a number of times also for sending some IOTA to lacksfish. (I read somewhere that this will be automated in the next release of the wallet.) I hope that helps. I don't feel confidant in what I am doing but I am making some progress. Maybe someone else can provide a better answer.
